docs/faq.mdx

```diff
@@ -56,7 +56,7 @@ Currently, registration costs are set at the following prices:
 
 3 and 4 character names have higher pricing to reflect the small number of these names available.
 
-To read more about the pricing structure of .eth names [read more about pricing](/registry/eth)
+To read more about the pricing structure of .eth names [read more about the pricing](/registry/eth)
```

Comment:
This sentence didn't really make sense, even before your change 😅. I think we should just delete it and add a link to the line above.
```suggestion
```